docs/index.html

```diff
@@ -0,0 +1,89 @@
+          </g>
+        </svg>
+        Mac Free Download</a>
+      <p class="under-button"><span class="under-button-span">Download for other platforms.</span> No affiliation with YouTube.</p>
+    </section>
+
+    <section class="second">
+      <h1>OSX, Windows and Linux</h1>
+      <div class="image-box">
+        <img class="first-img" src="./images/home_page.png" alt="">
+        <img style="height:440px" class="second-img" src="./images/search_page.png" alt="">
+        <img class="third-img" src="./images/video_page.png" alt="">
+      <div class="image-box">
+    </section>
+
+    <section class="third">
+      <h1>All the features you're used to</h1>
+      <ul>
+        <li>
+          <p>Video Search</p>
+          <p>Playback</p>
+        </li>
+        <li>
+          <p>Suggested Videos</p>
+          <p>Trending Videos</p>
+        </li>
+        <li>
+          <p>Videos by Category</p>
+          <p>Comment Feed</p>
+        </li>
+      </ul>
+    </section>
+
+    <section class="forth">
+      <h1>Featured Technologies</h1>
+        <ul>
+          <li>
+            <p>Electron</p>
+            <p>Integration</p>
+          </li>
+          <li>
+            <p>React.js</p>
+            <p>Framework</p>
+          </li>
+          <li>
+            <p>Webpack</p>
+            <p>Codesplitting</p>
+          </li>
+        </ul>
+    </section>
+
```

Comment:
we need a section somewhere that includes the download links for all the different platforms, just in case the dynamic button doesn't work properly.